.github/workflows/publish-sdk.yml

```diff
@@ -35,12 +35,13 @@ jobs:
             echo "Variable BRANCH_NAME is set to 'development'"
             yarn run change:tag
           fi
+     
```

Comment:
IMO should be
```yaml
      - name: Check branch name
        run: echo ${GITHUB_REF#refs/*/}
      - run: yarn install
      - run: |
          if [[ ${GITHUB_REF#refs/*/} == "development" ]]; then
            echo "Variable BRANCH_NAME is set to 'development'"
            yarn run change:tag
          fi
      - run: yarn run release
        env:
          GITHUB_TOKEN: ${{secrets.MY_GITHUB_PERSONAL_ACCESS_TOKEN}}
          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
          ```

Reply:
also need to remove the     `"remove:tag": "node removeDevelopmentTag.js"` script from `package-lock.json` and the js file